ERCS/erc-7802.md

```diff
@@ -0,0 +1,259 @@
+---
+eip: 7802
+title: Crosschain Token Interface
+description: Token interface that allows authorized contracts to mint and burn token representations during cross-chain transfers
```

Comment:
Similarly, you can remove some nonspecific bits from your description (and hopefully expand it further):

```suggestion
description: Allow authorized contracts to mint and burn token representations during cross-chain transfers
```